### ISSUE TITLE
Issue with upgrade to 1.14.x

### ISSUE THREAD

**Issue by 0x3e4** [seatsurfing/backend#195]
Hey everyone,

im still encountering an issue while trying to upgrade to version 1.14.x. I have added the FRONTEND_URL environment variable to the services as mentioned in the 1.14.1 release, but im getting "Secure Connection Failed" errors on versions 1.14.2 and above across different browsers.

My instance is hosted behind a NetScaler, and 1.14.1 is the latest version that seems to work (though im facing a language issue here #166 / #171). The latest stable version is still 1.13.8, which runs without any issues. The Docker logs dont show any interesting outputs, but I suspect that there might be a missing debug logging.

If youd like me to investigate further into the containers, please let me know where i should focus my attention in more detail.

Thank you all in advance!

**Comment by virtualzone** [comment]
Hey @0x3e4, as Seatsurfing does not offer secure connections, I guess there's an issue with your Netscaler configuration here (seems like it's trying to establish a secure connection, which is not possible).

**Comment by 0x3e4** [comment]
but it actually works without any issues when running all containers on 1.13.8? Service is binded on HTTP in the NetScaler config but i dont think that the reverse proxy is the issue here.. hmmm

running backend:1.14.4 and booking-ui + admin-ui on 1.13.8 also works without any issues.. i have just problems when updaing booking-ui + admin-ui to 1.14.x

maybe any logs i can check inside of the booking-ui/admin-ui containers?

**Comment by virtualzone** [comment]
Hi @0x3e4, please try a new setup without any additional configuration and without your Netscaler instance as a reverse proxy. Please use the following ```docker-compose.yml``` from the Readme (just changed all URLs to localhost:8080).

When accessing the app via http://localhost:8080, does the issue still occur?

```yaml
version: '3.7'

services:
  server:
    image: seatsurfing/backend
    restart: always
    networks:
      sql:
      http:
    ports:
      - 8080:8080
    environment:
      POSTGRES_URL: 'postgres://seatsurfing:DB_PASSWORD@db/seatsurfing?sslmode=disable'
      JWT_SIGNING_KEY: 'some_random_string'
      BOOKING_UI_BACKEND: 'booking-ui:3001'
      ADMIN_UI_BACKEND: 'admin-ui:3000'
      PUBLIC_URL: 'http://localhost:8080'
      FRONTEND_URL: 'http://localhost:8080'
  booking-ui:
    image: seatsurfing/booking-ui
    restart: always
    networks:
      http:
    environment:
      FRONTEND_URL: 'http://localhost:8080'
  admin-ui:
    image: seatsurfing/admin-ui
    restart: always
    networks:
      http:
    environment:
      FRONTEND_URL: 'http://localhost:8080'
  db:
    image: postgres:12
    restart: always
    networks:
      sql:
    volumes:
      - db:/var/lib/postgresql/data
    environment:
      POSTGRES_PASSWORD: DB_PASSWORD
      POSTGRES_USER: seatsurfing
      POSTGRES_DB: seatsurfing

volumes:
  db:

networks:
  sql:
  http:
```

**Comment by 0x3e4** [comment]
i think this is fixed with 1.19.5.. i could without any issues update my booking-ui and admin-ui to this version.
